cg/meta/workflow/rnafusion.py

```diff
@@ -0,0 +1,299 @@
+        all: bool,
+        pizzly: bool,
+        squid: bool,
+        starfusion: bool,
+        fusioncatcher: bool,
+        arriba: bool,
+        dry_run: bool = False,
+    ) -> None:
+        """Execute RNAFUSION run analysis with given options"""
+
+        options = self.__build_command_str(
+            self.get_verified_arguments(
+                case_id=case_id,
+                work_dir=work_dir,
+                resume=resume,
+                profile=profile,
+                with_tower=with_tower,
+                stub=stub,
+                input=input,
+                outdir=outdir,
+                genomes_base=genomes_base,
+                trim=trim,
+                fusioninspector_filter=fusioninspector_filter,
+                all=all,
+                pizzly=pizzly,
+                squid=squid,
+                starfusion=starfusion,
+                fusioncatcher=fusioncatcher,
+                arriba=arriba,
+            )
+        )
+        nextflow_options = self.__build_command_str(
+            self.get_verified_arguments_nextflow(
+                case_id=case_id,
+                log=log,
+            )
+        )
+        command = ["run", self.nfcore_pipeline_path]
+        parameters = (
+            nextflow_options
+            + ["-bg", "-q"]
+            + command
+            + options
+            + [
+                " > "
+                + str(self.get_case_path(case_id))
+                + "/"
+                + case_id
+                + "-stdout.log 2> "
+                + str(self.get_case_path(case_id))
+                + "/"
+                + case_id
+                + "-stdout.err  < /dev/null & "
+            ]
+        )
+        self.process.run_command(parameters=parameters, dry_run=dry_run)
+
+    def get_cases_to_analyze(self) -> List[models.Family]:
+        cases_query: List[models.Family] = self.status_db.cases_to_analyze(
+            pipeline=self.pipeline, threshold=self.threshold_reads
+        )
+        cases_to_analyze = []
+        for case_obj in cases_query:
+            if case_obj.action == "analyze" or not case_obj.latest_analyzed:
+                cases_to_analyze.append(case_obj)
+        return cases_to_analyze
+
+    def get_deliverables_file_path(self, case_id: str) -> Path:
+        """Returns a path where the rnafusion deliverables file for the case_id should be located.
+
+        (Optional) Checks if deliverables file exists
+        """
+        return Path(self.get_case_path(case_id), case_id + "_deliverables.yaml")
+
+    def report_deliver(self, case_id: str) -> None:
+        """Write report deliver"""
+        df = pd.read_csv(resources.rnafusion_bundle_filenames_path)
+        df = df.replace({"PATHTOCASE": str(self.get_case_path(case_id))}, regex=True)
+        df = df.replace({"CASEID": case_id}, regex=True)
+        df["path_index"] = "~"
+        text = yaml.dump(df.to_dict(orient="records")).replace("'~'", "~")
```

Comment:
```suggestion
        content = yaml.dump(df.to_dict(orient="records")).replace("'~'", "~")
```
More agnostic and here you should use the io/controller instead. Type hint

Reply:
Outdate due to [619adaa](https://github.com/Clinical-Genomics/cg/pull/1541/commits/619adaa454ebdebf0a71e9905806fe4031d469e2), see https://github.com/Clinical-Genomics/cg/pull/1541#discussion_r959695241, marking as resolved